encorelab/SCORE-API
# chore(Code): Merge WISE into SCORE on 2023-10-27

## Changes

Merged WISE into SCORE which includes new WISE code from [2023-09-28 to 2023-10-27](https://github.com/WISE-Community/WISE-API/compare/3707067faa0413878dfebffeb325298d3a6fec4d...a3f28b1c94dd6d6f1e8c2907dd74d9a7b0da9dea)

## Test

- Test with https://github.com/encorelab/SCORE-Client/pull/37
- Make sure the new code from WISE works
- Make sure the existing SCORE code works
